server/views.py

```diff
@@ -51,7 +59,7 @@ def user(request, name=None):
                 'title': v[1],
                 'last_revision': NotebookRevision.objects
                 .filter(notebook_id=v[0]).last().created.isoformat(sep=' ')
```

Comment:
I think this is going to cause a db-lookup-per-notebook, which will be slow. I suspect you can just use the annotation above with some modification.

Reply:
Good point!

Reply:
fwiw I think we noted that this would be an issue way back when we put this view together